chia/data_layer/download_data.py

```diff
@@ -146,11 +146,25 @@ async def insert_from_delta_file(
             async with aiohttp.ClientSession() as session:
                 async with session.get(server_info.url + "/" + filename, timeout=timeout) as resp:
                     resp.raise_for_status()
-
+                    size = int(resp.headers.get("content-length", 0))
+                    log.debug(f"Downloading delta file {filename}. Size {size} bytes.")
+                    progress_byte = 0
+                    progress_percentage = "{:.0%}".format(0)
                     target_filename = client_foldername.joinpath(filename)
-                    text = await resp.read()
-                    target_filename.write_bytes(text)
+                    with target_filename.open(mode="wb") as f:
+                        async for chunk, end_of_http_chunk in resp.content.iter_chunks():
+                            f.write(chunk)
+                            if log.isEnabledFor(logging.INFO):
```

Comment:
we don't use this anywhere in the code: `if log.isEnabledFor(logging.INFO):`. could you remove it, to keep the code consistency?